### ISSUE TITLE
Func code for the smart-contracts

### ISSUE THREAD

**Issue by NoelJacob** [toncenter/tonweb#42]
Add a link or a folder for all the func code.
https://github.com/toncenter/tonweb/search?q=oneFromBoc

**Comment by EmelyanenkoK** [comment]
Don't get it. Do you mean something like that? https://github.com/toncenter/tonweb/blob/9a10ecb1b5c703a3115c1d65f708c3ff68f2188d/src/contract/wallet/WalletSources.md

**Comment by NoelJacob** [comment]
Yeah exactly func/fift for that. I was specifically looking at NFT contracts.
